arangod/Utils/SupportInfoBuilder.cpp

```diff
@@ -0,0 +1,766 @@
+          result.openObject();
+          size_t numShards = coll->numberOfShards();
+          result.add("n_shards", VPackValue(numShards));
+          result.add("rep_factor", VPackValue(coll->replicationFactor()));
+
+          auto ctx = transaction::StandaloneContext::Create(*vocbase);
+
+          auto& collName = coll->name();
+          result.add("name", VPackValue(collName));
+          size_t planId = coll->planId().id();
+          result.add("plan_id", VPackValue(planId));
+
+          SingleCollectionTransaction trx(ctx, collName,
+                                          AccessMode::Type::READ);
+
+          Result res = trx.begin();
+
+          if (res.ok()) {
+            OperationOptions options(ExecContext::current());
+
+            OperationResult opResult =
+                trx.count(collName, transaction::CountType::Normal, options);
+            std::ignore = trx.finish(opResult.result);
+            if (opResult.fail()) {
+              LOG_TOPIC("8ae00", WARN, Logger::STATISTICS)
+                  << "Failed to get number of documents: "
+                  << res.errorMessage();
+            } else {
+              result.add("n_docs", VPackSlice(opResult.buffer->data()));
+            }
+
+          } else {
+            LOG_TOPIC("e7497", WARN, Logger::STATISTICS)
+                << "Failed to begin transaction for getting number of "
+                   "documents: "
+                << res.errorMessage();
+          }
+
+          if (collsAlreadyVisited.find(planId) == collsAlreadyVisited.end()) {
+            auto collType = coll->type();
+            if (collType == TRI_COL_TYPE_DOCUMENT) {
+              numDocColls++;
+              result.add("type", VPackValue("document"));
+            } else if (collType == TRI_COL_TYPE_EDGE) {
+              result.add("type", VPackValue("edge"));
+              numEdgeColls++;
+            } else {
+              result.add("type", VPackValue("unknown"));
+            }
+            if (coll->isSmart()) {
+              numSmartColls++;
+              result.add("smart_graph", VPackValue(true));
+            } else {
+              result.add("smart_graph", VPackValue(false));
+            }
+            if (coll->isDisjoint()) {
+              numDisjointSmartColls++;
+              result.add("disjoint", VPackValue(true));
+            } else {
+              result.add("disjoint", VPackValue(false));
+            }
+
+            containers::FlatHashMap<std::string, size_t> idxTypesToAmounts;
+
+            auto flags = Index::makeFlags(Index::Serialize::Estimates,
+                                          Index::Serialize::Figures);
+            std::vector<std::string_view> idxTypes = {
+                "edge",      "geo",        "hash",      "fulltext", "inverted",
+                "no-access", "persistent", "iresearch", "skiplist", "ttl",
+                "zkd",       "primary",    "unknown"};
```

Comment:
I don't think that an index of type "no-access" actually exists. Where is the type from?
I think older deployments can also have old indexes of type "geo1" and "geo2". These should be mapped to "geo" simply.